docs/howto/python/preinstalled_packages.md

```diff
@@ -0,0 +1,9 @@
+# Pre-install additional packages with emscripten-forge
```

Comment:
Since most of the other guides in this category are about the Pyolite kernel, maybe we should make it more explicit that this will only work with xeus python:

![image](https://user-images.githubusercontent.com/591645/209347341-8077a1e0-3856-4f93-8044-026bf84062c8.png)

Reply:
Also maybe a small `warning` at the top of this file to make it more visible this is for a different kernel.